src/app/js/formats/shared/LodexResource.js

```diff
@@ -35,22 +35,46 @@ const styles = StyleSheet.create({
 });
 
 // see https://jsonfeed.org/version/1#items
-const LodexResource = ({ id, title, summary }) => {
+const LodexResource = ({ id, url, title, summary }) => {
     if (!id) {
         return null;
     }
-    return (
-        <div id={id}>
-            <Link
-                className={css(styles.contentLink)}
-                to={getFullResourceUri({ uri: id })}
-            >
+    let urlDomain = url.replace(id, '');
+    let getFullResourceUriDomain = getFullResourceUri({ uri: id }).replace(
```

Comment:
This variable is a string, not a function, it should not contain a verb (`get`).
I would rename it to `fullResourceUriDomain`.